main/java/BeforeAfter.java

```diff
@@ -0,0 +1,69 @@
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import shop.Cart;
+import parser.JsonParser;
+import shop.RealItem;
+import shop.VirtualItem;
+
+import java.io.File;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class BeforeAfter {
+    Cart newCart = null;
+    JsonParser parser = null;
+    RealItem car = null;
+    VirtualItem disk = null;
+
+    @BeforeEach
+    public Cart createCart() {
+        newCart = new Cart("new-cart");
+        System.out.println(newCart.toString() + " Cart is created");
+        return newCart;
+    }
+
+    @BeforeEach
+    public JsonParser createJsonParser() {
+        parser = new JsonParser();
+        System.out.println(parser.toString() + " is created");
+        return parser;
+    }
+
+    @BeforeEach
+    public RealItem createRealItemCar() {
+        car = new RealItem();
+        System.out.println(car.toString() + " is created");
+        return car;
+    }
+
+    @BeforeEach
+    public VirtualItem createVirtualItemDisk() {
+        disk = new VirtualItem();
+        System.out.println(disk.toString() + " is created");
+        return disk;
+    }
+
+    @AfterEach
+    public void removeAllFilesCreatedInTests() {
+        System.out.println("All Files created by Test is going to be removed");
+        File folder = new File("src/main/resources/");
+        File[] arrayListOfFiles = folder.listFiles();
+        ArrayList<File> arrayListListOfFiles = new ArrayList<File>(Arrays.asList(arrayListOfFiles));
```

Comment:
Lines 52-67 simplify to:
if (arrayListOfFiles != null) {
            Stream.of(arrayListOfFiles)
                    .filter(file -> !file.getName().contains("andrew-cart.json") && !file.getName().contains("eugen-cart.json"))
                    .forEach(f -> new File(f.getAbsolutePath()).delete());
        }

Reply:
Correction is applied.